stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/TextFieldController.kt

```diff
@@ -105,7 +106,8 @@ class SimpleTextFieldController constructor(
     override val visualTransformation =
         textFieldConfig.visualTransformation ?: VisualTransformation.None
 
-    override val label: Flow<Int> = MutableStateFlow(textFieldConfig.label)
+    override val label = MutableStateFlow(textFieldConfig.label)
+    override val placeHolder: Flow<String?> = MutableStateFlow(textFieldConfig.placeHolder)
```

Comment:
Does this need to be a flow? Are there scenarios where we want placeHolder to be dynamic?

Reply:
Hmm for the usecase we need in identity it doesn't, but just added it to align with other fields which are all defined as flow here